test/docfx.RegressionTest/RegressionTest.cs

```diff
@@ -90,6 +90,17 @@ static string GetDocfxConfig(Options opts)
                     docfxConfig["outputUrlType"] = "ugly";
                     docfxConfig["template"] = "https://github.com/Microsoft/templates.docs.msft.pdf#master";
                 }
+
+                if (opts.RegressionMarkdownRule)
+                {
+                    docfxConfig["markdownValidationRules"] = "https://ops/regressionallcontentrules/";
```

Comment:
why not set the actual URL directly here?

Reply:
the actual url is enabled AuthN

Reply:
ok